server/src/main/java/org/apache/druid/segment/indexing/RealtimeTuningConfig.java

```diff
@@ -46,8 +42,6 @@ public class RealtimeTuningConfig implements AppenderatorConfig
 {
   private static final Period DEFAULT_INTERMEDIATE_PERSIST_PERIOD = new Period("PT10M");
```

Comment:
I think most of these constants have been moved to `SeekableStreamIndexTaskTuningConfig`. Do we still need this class?

Reply:
Ah, no, we don't need it. I went through all the trouble of adjusting all the call sites to not use this, but then forgot to delete it. Fixed!